scripts/dev/newReleaseBranch.sh

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_DIR="${SCRIPT_DIR}/../../.."
+
+usage() {
+    cat << ???  1>&2;
+
+Creates HotFix and ServicePack (.N) branches for repo, share and packaging projects, and updates the master
+or ServicePack branches ready for the next release.
+
+Usage: $0 <hotFixVersion> [-v <masterVersion>] [-h] [-l]
+  <hotFixVersion>:  HotFix Branch to be created or modified.
```

Comment:
This is really the SP version. It's used for the HF branch name, but it's important that the parameter only has three parts.  e.g. Passing in 23.1.0.1 will break the script (or just cause a failure maybe - I didn't check).

Reply:
I think you are correct to identify <hotFixVersion> in the Usage message could be improved. Changing it to:

Usage: $0 <releasedVersion> [-v <masterVersion>] [-h] [-l]
  <releasedVersion>:  The version just released. Used in the new HotFix Branch name release/<releasedVersion>.
                                    Must contain 3 integers separated by dots.

Internally the value is still assigned to the hotFixVersion variable as that still make sense.